app/interactors/notify_about_checkin.rb

```diff
@@ -5,6 +5,30 @@ class NotifyAboutCheckin
 
   def call
     device.notify_subscribers("new_checkin", checkin)
-    device.broadcast_checkin_for_friends(checkin)
+    broadcast_checkin_for_friends
+  end
+
+  private
+
+  def broadcast_checkin_for_friends
+    device.user.friends.find_each do |friend|
+      next unless ConnectedList.all.include?(friend.id.to_s) && broadcast_checkin?(friend)
```

Comment:
what will happen here without `ConnectedList` check? (if sending messages to user who is not currently connected)